src/game_api/memory.cpp

```diff
@@ -103,6 +103,7 @@ LPVOID alloc_mem_rel32(size_t addr, size_t size)
 }
 
 std::unordered_map<std::string, std::vector<RecoverableMemory>> original_memory;
+std::unordered_set<std::string> edited_memory;
```

Comment:
Feel like better solution would be to change `std::vector<RecoverableMemory>` into a struct with vector and bool

Reply:
You're